health/REFERENCE.md

```diff
@@ -370,6 +371,49 @@ increasing. Eventually, the comparison will find the averages in the two time-fr
 However, the issue was not resolved, it's just a matter of the newer data "polluting" the old. For such alarms, it's a
 good idea to tell Netdata to not clear the notification, by using the `no-clear-notification` option.
 
+#### Alarm line `label`
+
+Defines the list of labels expected on host, for example, let us suppose that `netdata.conf` is configured with the
+following labels:
+
+```yaml
+[host labels]
+    installed = 20191211
+    room = server
+```
+
+but, we also have for the workstations the following `netdata.conf`
+
+```yaml
+[host labels]
+    installed = 201705
+    room = workstation
+```
+
+I have defined inside `netdata.conf` labels to classify hosts and now we can apply alarms for them using labels, for example,
+case I add the following line inside alarms:
+
+```yaml
+label: room = server
+```
+
+this will apply alarms only on hosts that have `room = server`. 
+It is also possible to combine labels to apply alarm, for example, case I want to raise a specific alarm only for hosts
+inside a room that were installed in a specific time, I can write the following label condition:
```

Comment:
You can also combine labels when applying them to alarms. For example, if you want to raise a specific alarm only for hosts inside a room that were installed at a specific time, you can write the following label line: